src/Page/HelpSelfSingle.elm

```diff
@@ -35,29 +34,130 @@ view : String -> Model -> Html msg
 view category model =
     let
         categoryData =
-            getCategoryStrings category
+            getCategoryKeys category
     in
     div []
         [ header []
             [ h1 [ css [ pageHeadingStyle ] ] [ text (t categoryData.title) ]
+            , case categoryData.resources of
+                Just resources ->
+                    div [] [ renderResources resources ]
+
+                Nothing ->
+                    text ""
             ]
         , ul []
-            [ li [] [ a [ href (t GetHelpPageSlug) ] [ text "Get Help" ] ]
-            , li [] [ a [ href (t NotAlonePageSlug) ] [ text "Read about others" ] ]
-            , li [] [ a [ href (t DefinitionPageSlug) ] [ text "What is Economic Abuse" ] ]
+            [ li [] [ a [ href ("../" ++ t HelpSelfGridPageSlug) ] [ text (t ToHelpSelfFromSingleCategoryLink) ] ]
             ]
         ]
 
 
-type alias CategoryData =
-    { title : Key }
+renderResources : List CategoryResource -> Html msg
+renderResources resources =
+    div []
+        (List.map
+            (\resource ->
+                div []
+                    [ p [] [ text (t resource.title) ]
+                    , renderQuotes resource.quotes
+                    , p [] [ text (t resource.summary) ]
+                    , div [] [ a [ href (t resource.linkHref) ] [ text (t resource.linkName) ] ]
+                    ]
+            )
+            resources
+        )
+
 
+renderQuotes : List Key -> Html msg
+renderQuotes quoteKeys =
+    div [] (List.map (\quoteKey -> p [] [ text (t quoteKey) ]) quoteKeys)
 
-getCategoryStrings : String -> CategoryData
-getCategoryStrings category =
-    case categoryFromString category of
-        COVID ->
-            { title = HelpSelfCategory1Title }
 
-        _ ->
-            { title = HelpSelfCategory2Title }
+type alias CategoryResource =
+    { title : Key
+    , quotes : List Key
+    , summary : Key
+    , linkName : Key
+    , linkHref : Key
+    }
+
+
+type alias CategoryData =
+    { title : Key
+    , resources : Maybe (List CategoryResource)
+    }
+
+
+getCategoryKeys : String -> CategoryData
+getCategoryKeys category =
+    if category == t HelpSelfCategory1Slug then
+        { title = HelpSelfCategory1Title
+        , resources =
+            Just
+                [ { title = HelpSelfCategory1Resource1Title
+                  , quotes = [ HelpSelfCategory1Resource1Quote1, HelpSelfCategory1Resource1Quote2 ]
+                  , summary = HelpSelfCategory1Resource1Summary
+                  , linkName = HelpSelfCategory1Resource1Link
+                  , linkHref = HelpSelfCategory1Resource1Href
+                  }
+                ]
+        }
+
+    else if category == t HelpSelfCategory2Slug then
+        { title = HelpSelfCategory2Title
+        , resources =
+            Just
+                [ { title = HelpSelfCategory2Resource1Title
+                  , quotes = [ HelpSelfCategory2Resource1Quote1, HelpSelfCategory2Resource1Quote2 ]
```

Comment:
I'm guessing this won't change often enough to worry about, but it might be easier if text.elm could take a list.